content/tutorials/manage-the-rippled-server/installation/build-run-rippled-ubuntu.md

```diff
@@ -30,7 +30,7 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install `ripple
 
 3. Install dependencies.
 
-        sudo apt-get -y install git pkg-config protobuf-compiler libprotobuf-dev libssl-dev wget build-essential
+        sudo apt-get -y install git pkg-config protobuf-compiler libprotobuf-dev libssl-dev wget build-essential doxygen
```

Comment:
I guess this is OK. Doxygen is optional, and only used for some internal docs, but I guess those may be useful to contributors who are dedicated enough to build from source.